preupg/application.py

```diff
@@ -35,10 +34,12 @@
 
 
 def fault_repr(self):
-    """monkey patching Fault's repr method so newlines are actually interpreted"""
+    """monkey patching Fault's repr method so newlines are actually interpreted
+    """
```

Comment:
That's against documentation convention (https://www.python.org/dev/peps/pep-0257/)